src/Console/Commands/MvcContractsUpdate.php

```diff
@@ -55,8 +55,8 @@ public function handle(DocumentSignInterface $documentSign)
                     $contract->status = Contract::STATUS_DENIED;
                     break;
                 case 'si':
-                    $contract->signed_at = Carbon::createFromTimestamp(strtotime($signRequest['signed_on']), 'Europe/Amsterdam');
                     $contract->status = Contract::STATUS_SIGNED;
+                    $contract->signers = $signRequest['signers'];
```

Comment:
I dont like the name $signRequest because this is the name of one document sign service use something like: 
$documentSignature.

Also it is not save to say 'signers' is always present so use Arr:get for that.